script/plugin_webpack_directories.rb

```diff
@@ -39,8 +39,10 @@ def entry_name(plugin_name, entry)
 specs.each do |dep|
   # skip other rails engines that are not plugins
   # TODO: Consider using the plugin registration api?
-  next unless dep.name =~ plugin_name_regexp
-  next if dep.name =~ /.*[_-]core$/
+  unless dep.metadata.has_key?("is_foreman_plugin") && dep.metadata["is_foreman_plugin"] == "true"
```

Comment:
Wouldn't it be easier to use:
```suggestion
  unless dep.metadata["is_foreman_plugin"]
```

This would require using real booleans rather than quoting and I don't know if you can in Rubygem metadata.

Reply:
I would also prefer to have something like s.metadata = { "foreman" => { "plugin" => "yes", "more_data" } } .. but:

    The validation error was 'metadata values must be a String'


So, it can only be { "key" => "value" }

Reply:
That's too bad.